index.bs

```diff
@@ -1384,14 +1394,14 @@ with a given ([=aggregatable debug rate-limit record/context site=], [=aggregata
 per [=aggregatable debug rate-limit window=]. Its value is (2<sup>20</sup>, 65536).
 
 <dfn>Default max event states</dfn> is a positive integer that controls the
```

Comment:
Is the concept of default max event states even necessary anymore? We might be better off requiring the `max_event_states` field to be given an explicit value rather than assume that the surrounding registration uses the default event configuration.

Reply:
Can we address this in a separate PR? was hoping to make this PR a pure refactor so that we can check it in sooner and we can move forward with the other 2 PRs (info gain, source processing).

Reply:
Synced offline. Let's keep this as-is for now, filed b/359253451 to track this.